OneSignalSDK/unittest/src/test/java/com/onesignal/ShadowOSUtils.java

```diff
@@ -32,21 +32,36 @@ public static boolean isGMSInstalledAndEnabled() {
 
    // Huawei: HMS
    public static boolean hasHMSAvailability;
-   public boolean hasHMSAvailability() {
+   public static boolean hasHMSAvailability() {
       return hasHMSAvailability;
    }
    public static boolean hasHMSPushKitLibrary;
-   public boolean hasHMSPushKitLibrary() {
+   public static boolean hasHMSPushKitLibrary() {
       return hasHMSPushKitLibrary;
    }
+   public static boolean hasHMSAGConnectLibrary;
+   public static boolean hasHMSAGConnectLibrary() {
+      return hasHMSAGConnectLibrary;
+   }
+
    public static boolean isHMSCoreInstalledAndEnabled;
    public static boolean isHMSCoreInstalledAndEnabled() {
       return isHMSCoreInstalledAndEnabled;
    }
+   public static boolean isHMSCoreInstalledAndEnabledFallback() {
+      return isHMSCoreInstalledAndEnabled;
+   }
+
+   public static void hasAllRecommendHMSLibraries(boolean value) {
```

Comment:
Add past tense `hasAllRecommendHMSLibraries` to `hasAllRecommendedHMSLibraries`?